src/Storage/Storage.Management/Storage.Management.csproj

```diff
@@ -14,7 +14,7 @@
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="9.1.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="10.0.0" />
```

Comment:
For a stable build, this package should be published

Reply:
I will update it to released one.

Reply:
Updated, by remove the private package in LocalFeed. 
Since the private and released one has same version, project file don't need change.